.travis.yml

```diff
@@ -54,6 +54,29 @@ matrix:
       services:
         - mysql
         - postgresql
+    # have a separate branch for javascript tests
+    - language: node_js
+      node_js: 6
+      dist: trusty
+      # overwrite php related settings
+      php:
+      services:
+      addons:
+      install:
+        - travis_retry npm install
+        # disable xdebug for performance in composer
+        - phpenv config-rm xdebug.ini || echo "xdebug is not installed"
+        - travis_retry composer self-update && composer --version
+        - travis_retry composer global require "fxp/composer-asset-plugin:^1.2.0"
```

Comment:
I'd add `--no-plugins` here for better stability when either composer or fxp updates.